src/resolvers/records.js

```diff
@@ -67,6 +67,16 @@ module.exports = {
 	Mutation: {
 		createRecord: async (parent, { domainId, input }, { req }) => {
 
+			// In case of own site, return fake entry and don't track
+			if (req.cookies && req.cookies.ackee_login === "1") {
+				return {
+					success: true,
+					payload: {
+						"id": "88888888-8888-8888-8888-888888888888"
```

Comment:
Please run `npm run lint` to align the code with the rest.

Reply:
Fixed here and in other files in this PR